apps/adgerdir/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -20,11 +23,22 @@ const useImageLoader = (url: string): boolean => {
   return loaded
 }
 
+const defaultBoxProps = {
+  alignItems: 'center',
+  width: 'full',
+  height: 'full',
+  display: 'inlineFlex',
+  overflow: 'hidden',
+  borderRadius: 'large',
+} as BoxProps
```

Comment:
Isn't it more natural to write `defaultBoxProps: BoxProps = { ... }` rather than typecasting?